gridstatus/ercot_api/ercot_api.py

```diff
@@ -1715,7 +1716,8 @@ def get_historical_data(
             files = self._individually_download_documents(links=links, verbose=verbose)
 
         if not read_as_csv:
-            return files
+            # Only return the bytes (not the filenames)
+            return [f[0] for f in files]
```

Comment:
The `get_60_day_sced_disclosure` and `get_60_day_dam_disclosure` methods on `ErcotAPI` were broken and this change gets them working again. Those are the only methods that call `get_historical_data` with `read_as_csv=False` so were the only methods impacted.